bot/exts/moderation/incidents.py

```diff
@@ -402,10 +515,106 @@ async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent) ->
 
     @Cog.listener()
     async def on_message(self, message: discord.Message) -> None:
-        """Pass `message` to `add_signals` if and only if it satisfies `is_incident`."""
+        """
+        Pass `message` to `add_signals` and `extract_message_links` if it satisfies `is_incident`.
+
+        If the message (`message`) is an incident report, then run it through `extract_message_links`
+        to get all the message link embeds (embeds which contain information about that particular
+        link).These message link embeds are then sent into the channel.
```

Comment:
Why not just "If `message` is an incident report, ..."

Mind the missing space after the dot on the L523.

Reply:
Resolved in 88cefa6.